sycl/test-e2e/NewOffloadDriver/buffer.cpp

```diff
@@ -0,0 +1,58 @@
+// REQUIRES: level_zero
+// RUN: %clangxx -fsycl --offload-new-driver %s -o %t.out
+// RUN: %{run} %t.out
```

Comment:
These should be below the copyright header

Reply:
imo this pr has been through a lot of iteration already, we should resolve this question in general for all tests outside of this pr and then make a changing implementing the rule (and ideally a CI check to enforce it in the future